pkg/config/config.go

```diff
@@ -59,6 +59,11 @@ type Server struct {
 
 	// currently only used in our demo app
 	HideApplications []string `def:"" desc:"please don't use, this will soon be deprecated"`
+
+	OutOfSpaceThreshold              float64 `def:"10" desc:"Threshold value to consider out of space in percentage"`
+	OutOfSpaceWarningThreshold       float64 `def:"20" desc:"Threshold value to consider to show out of space in warning percentage"`
+	OutOfSpaceStaticThreshold        int     `def:"512" desc:"Threshold static value to consider out of space. Useful when % value is very high than required value"`
+	OutOfSpaceWarningStaticThreshold int     `def:"1024" desc:"Threshold static value to consider to show out of space warning. Useful when % value is very high than required value"`
```

Comment:
I know we've decided earlier that it makes sense to have 4 values, but I thought about it some more (also talked to @Rperry2174) and I think we should just leave 1 and have it be the threshold value in bytes for when the writes should stop. We can set it at 512 MB and I will run some tests later to determine a perfect value for this. This will make code simpler and it will be easier for users to understand what this is (currently it's really hard to get the difference between the 4 values). And it will work on any system. And if users have too little space allocated for pyroscope this way we will be able to notify them about it.

And, after we implement #168 (data retention) we can set the whole thing up in a way that it would automatically delete old data when it's above the threshold. This way we also don't need the warning thresholds.

Sorry about the confusion.